### PR TITLE
next: prep for `@next` release

### PR DESCRIPTION
**Pull request by huntabyte**

### CODE REVIEW

**Comment by cowboycodr** [comment]
oh-my-god-its-happening

File: packages/cli/src/utils/get-config.ts [hunk 1]
```diff
@@ -105,7 +106,7 @@ export function getTSConfig(cwd: string, tsconfigName: "tsconfig.json" | "jsconf
 	const parsedConfig = getTsconfig(path.resolve(cwd, "package.json"), tsconfigName);
 	if (parsedConfig === null) {
 		throw error(
-			`Failed to find ${highlight(tsconfigName)}. See: ${color.underline("https://www.shadcn-svelte.com/docs/installation#opt-out-of-typescript")}`
+			`Failed to find ${highlight(tsconfigName)}. See: ${color.underline(`${SITE_BASE_URL}docs/installation#opt-out-of-typescript`)}`
```
Comment (AdrianGonz97) [inline packages/cli/src/utils/get-config.ts:109]:
```suggestion
			`Failed to find ${highlight(tsconfigName)}. See: ${color.underline(`${SITE_BASE_URL}/docs/installation#opt-out-of-typescript`)}`
```

**Comment by AdrianGonz97** [comment]
We should probably update all these spots on the site as well:
img

**Comment by huntabyte** [comment]
Closing this in favor of #1347 b/c lazy and don't want to resolve all the merge conflicts :)

**Pull request closed without merging**